packages/flutter/lib/src/material/about.dart

```diff
@@ -607,31 +607,30 @@ class _PackagesViewState extends State<_PackagesView> {
     final _LicenseData data,
     final bool drawSelection,
   ) {
-    return ListView(
-      children: <Widget>[
-        widget.about,
-        ...data.packages
-            .asMap()
-            .entries
-            .map<Widget>((MapEntry<int, String> entry) {
-              final String packageName = entry.value;
-              final int index = entry.key;
-              final List<int> bindings = data.packageLicenseBindings[packageName]!;
-              return _PackageListTile(
-                packageName: packageName,
-                index: index,
-                isSelected: drawSelection && entry.key == (selectedId ?? 0),
-                numberLicenses: bindings.length,
-                onTap: () {
-                  widget.selectedId.value = index;
-                  _MasterDetailFlow.of(context)!.openDetailPage(_DetailArguments(
-                    packageName,
-                    bindings.map((int i) => data.licenses[i]).toList(growable: false),
-                  ));
-                },
-              );
-            }),
-      ],
+    return ListView.builder(
+      itemCount: data.packages.length + 1,
+      itemBuilder: (BuildContext context, int index) {
+        if (index == 0) {
+          return widget.about;
+        }
+        // Decrement index by one since we show the about widget first
+        index -= 1;
```

Comment:
It's a little odd to modify the parameter of this closure. Maybe define a new variable `final int packageIndex = index - 1;` and use that below.

Reply:
Yes, agreed. 👍  Updated with that change and rebased on the latest master.